types/src/transaction/deploy/deploy_header.rs

```diff
@@ -111,7 +111,7 @@ impl DeployHeader {
         config: &TransactionConfig,
         timestamp_leeway: TimeDiff,
         at: Timestamp,
-        deploy_hash: &DeployHash,
+        deploy_hash: &TransactionHash,
```

Comment:
Should the argument be renamed, too?

Reply:
Good catch - it should actually stay as `DeployHash` since it's related to legacy deploy type. Updated in e225d848847bbc8715d6fef842c80e23ad08906f